visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts

```diff
@@ -194,7 +193,7 @@ export class CodeMapLabelService implements CameraChangeSubscriber {
 		this.previousScaling.copy(scaling)
 	}
 
-	onCameraChanged() {
+	onCameraChanged = () => {
```

Comment:
Can we make this method `private` now?

Reply:
`addEventListener(..., () => this.onFoo())` , `addEventListener(..., this.onFoo.bind(this))` and `foo = () => {}` are semantically the same here. `this` pointer in JS is weird. I just changed to previous way with next push for being consistent.

Reply:
just as remark: In `ViewCubeComponent` I kept `onCameraChanged = (data: { camera: PerspectiveCamera }) => { ... }`. Otherwise I would have to repeat the arguments when registering like `this.threeOrbitControlsService.subscribe("onCameraChanged", (data) => this.onCameraChanged(data))` and the others are registered the same way.